### PR TITLE
Small logging and configuration bug fixes for trigger

### PR DESCRIPTION
**Pull request by carlpulley-da**

### CODE REVIEW

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 1]
```diff
@@ -465,6 +465,8 @@ object Trigger {
 
   final class Version(protected val rank: Int) extends Ordered[Version] {
     override def compare(that: Version): Int = this.rank compare that.rank
+
+    override def toString: String = s"Version($rank)"
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:469]: Ensures that a meaningful trigger version is logged

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerConfig.scala [hunk 2]
```diff
@@ -69,7 +69,7 @@ object TriggerRunnerConfig {
       maximumBatchSize = 1000,
       batchingDuration = 250.milliseconds,
       hardLimit = TriggerRunnerHardLimits(
-        maximumActiveContracts = 10000,
+        maximumActiveContracts = 20000,
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerConfig.scala:72]: Fix to allow larger contract stores hard limits

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 3]
```diff
@@ -863,7 +865,7 @@ private[lf] class Runner private (
         .getTransactions(offset, None, filter)
         .map { transaction =>
           triggerContext.childSpan("update") { implicit triggerContext: TriggerLogContext =>
-            triggerContext.logDebug("Transaction source")
+            triggerContext.logDebug("Transaction source", "message" -> transaction)
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:868]: Logging a view of the received transaction message allows a ledger view of the ACS to be constructed should rule evaluation stall

File: triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala [hunk 4]
```diff
@@ -1291,24 +1293,34 @@ private[lf] class Runner private (
 
   private[this] def hideIrrelevantMsgs: TriggerContextualFlow[TriggerMsg, TriggerMsg, NotUsed] =
     TriggerContextualFlow[TriggerMsg].mapConcat[TriggerContext[TriggerMsg]] {
-      case ctx @ Ctx(_, msg @ TriggerMsg.Completion(c), _) =>
+      case ctx @ Ctx(_, TriggerMsg.Completion(c), _) =>
```
Comment (carlpulley-da) [inline triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala:1296]: Boy Scout work

**Review by akrmn — APPROVED** [review]

**Review by remyhaemmerle-da — APPROVED** [review]

**Review by remyhaemmerle-da — APPROVED** [review]

**Pull request merged**